e2e/swapSheetFlow.spec.js

```diff
@@ -241,6 +255,14 @@ describe('Swap Sheet Interaction Flow', () => {
     await Helpers.clearField('exchange-modal-input-native-246');
     await Helpers.checkIfVisible('exchange-modal-input');
     await Helpers.checkIfVisible('exchange-modal-output');
+  });
+
+  it('Should not show settings routes picker on Arbitrum or flashbots on L2s', async () => {
+    await Helpers.waitAndTap('exchange-settings-button');
+    await Helpers.checkIfVisible('swap-settings-header');
+    await Helpers.checkIfNotVisible('swap-settings-routes-label');
+    await Helpers.checkIfNotVisible('swap-settings-flashbots-label');
```

Comment:
this is nice, could we also add checks for flashbots not visible on optimism and polygon as well?